README.md

```diff
@@ -1 +1,75 @@
-# Stock-Market
\ No newline at end of file
+# Stock-Market
+
+## Deployment
+
+(Link will be provided after finishing the project)
+
```

Comment:
- [ ] Kindly make sure you deploy the application using Heroku, Netlify, or GitHub Pages the link to the deployed project is present in the README as required for the project.

Reply:
This issue is well resolved! ✔️